internal/service/ocm/ocm_impl.go

```diff
@@ -154,7 +194,31 @@ func createOrgAdminSearchString(users []models.User) string {
 	return search
 }
 
-func createQueryOrder(q models.UserQuery) string {
+func createAccountsV3UsersSearchString(orgID string) string {
+	return fmt.Sprintf("organization.id='%s'", orgID)
+}
+
+func createAccountsV3UsersBySearchString(orgID string, body models.UsersByBody) string {
+	search := ""
+
+	search += fmt.Sprintf("organization.id='%s'", orgID)
```

Comment:
rather than initializing to an empty string would you be ok with just assigning the sprintf directly? 

EDIT: looks like we could almost just call the createAccountsV3UsersSearchString function above.